cvmfs/object_fetcher.h

```diff
@@ -0,0 +1,387 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef OBJECT_FETCHER_H
+#define OBJECT_FETCHER_H
+
+#include <string>
+
+#include "manifest.h"
+#include "manifest_fetch.h"
+#include "history_sqlite.h"
+#include "catalog.h"
+
```

Comment:
The blank line can be removed.  In general it would be better to sort the includes alphabetically within each group (system includes, language includes, project includes) so that with many includes we can quickly find if a particular include exists or not.